### ISSUE TITLE
Cannot resolve dependency to assembly XAMLMarkupExtension because it is not preloaded...

### ISSUE THREAD

**Issue by ckocer** [SeriousM/WPFLocalizationExtension#61]
Hi there,
First of all, I'd like to thank you for that cool project but, I've a problem.

In my MainWindow XAML, Localizer works like a charm with that code:
xmlns:lex = "http://wpflocalizeextension.codeplex.com"
lex:LocalizeDictionary.DesignCulture="tr"
lex:ResxLocalizationProvider.DefaultAssembly="WPFLocalizeExtension"
lex:ResxLocalizationProvider.DefaultDictionary="Language"

I've a few windows and I want to use LocalizeExtension on them.
In my SettingsWindow, I've added the same code to XAML but when I try to build my solution, it gives:

Unknown build error, 'Cannot resolve dependency to assembly 'XAMLMarkupExtensions, Version=1.2.0.0, Culture=neutral, PublicKeyToken=null' because it has not been preloaded. When using the ReflectionOnly APIs, dependent assemblies must be pre-loaded or loaded on demand through the ReflectionOnlyAssemblyResolve event. 

this error.

All of my windows in the same project.
Thank you.

**Comment by SeriousM** [comment]
uhm, ähm... @MrCircuit, may you help with this please?

**Comment by MrCircuit** [comment]
Hi,

unfortunately, I'm not very familar with the deeper mechanics of NuGet. Try reinstalling the NuGet packages and make sure, that only version 1.2 of XamlMarkupExtensions is installed. If your project consists of several libraries containing your views, verify that each library has an appropriate reference to this package.

**Comment by indazoo** [comment]
I had the same error, maybe it helps you too or hints you in the right/wrong direction.
After adding the source code of wpfLocalizationExtension to my project I had this message.

Problem was the XAMLMarkupExtension is an external module and the dll was missing in the bin folder (Build output). 
I had added this project too, but then a century old VS Bug hit: It does not copy the dependent dll to my main project. I had to reference XAMLMarkupExtension project in my main project too. Maybe you are just missing XAMLMarkupExtension nUget(Dont'know exactly) I'm a total WPFLocalization newbie right now.
